inference-engine/tests_deprecated/functional/vpu/common/layers/myriad_layers_CTCDecoder_test.cpp

```diff
@@ -5,7 +5,7 @@
 #include "myriad_layers_CTCDecoder_test.hpp"
 
 INSTANTIATE_TEST_CASE_P(
-	accuracy, myriadCTCDecoderLayerTests_smoke,
+	DISABLED_accuracy, myriadCTCDecoderLayerTests_smoke,
```

Comment:
Why did you disable accuracy test?

Reply:
The test is incorrect as we agreed with VPU team. I created a ticket to track this bug: 43597